lnbits/extensions/boltcards/lnurl.py

```diff
@@ -88,13 +88,22 @@ async def api_scan(p, c, request: Request, external_id: str = None):
         return {"status": "ERROR", "reason": "Max daily limit spent."}
     hit = await create_hit(card.id, ip, agent, card.counter, ctr_int)
     lnurlpay = lnurl_encode(request.url_for("boltcards.lnurlp_response", hit_id=hit.id))
+
+    replacements = {
+        "refund_url": f"lnurl://{lnurlpay}",
+        "card_name": card.card_name,
+    }
+    message = card.message
+    for key, value in replacements.items():
+        message = message.replace("{" + key + "}", value)
+
     return {
         "tag": "withdrawRequest",
         "callback": request.url_for("boltcards.lnurl_callback", hitid=hit.id),
         "k1": hit.id,
         "minWithdrawable": 1 * 1000,
         "maxWithdrawable": card.tx_limit * 1000,
-        "defaultDescription": f"Boltcard (refund address lnurl://{lnurlpay})",
+        "defaultDescription": message,
```

Comment:
```suggestion
        "defaultDescription": card.message or f"Boltcard (refund address lnurl://{lnurlpay})",
```

Reply:
See reply above.

Reply:
Hey! Default values should not be hard coded into the migrations and in the model definition. Can you find an appropriate way to initialize those values?

Reply:
Accepted your suggestion.